public/admin.php

```diff
@@ -92,6 +98,43 @@
     return back()->withErrors(__('legacy.error.error'));
 }
 
+if ($action === 'migrate-achievement') {
+    $achievementIds = explode(',', requestInputSanitized('a'));
+    $gameId = requestInputSanitized('g');
+    if (!Game::where('ID', $gameId)->exists()) {
+        return back()->withErrors('Unknown game');
+    }
+
+    // determine which game(s) the achievements are coming from
+    $oldGames = Achievement::whereIn('ID', $achievementIds)->select(['GameID'])->distinct()->pluck('GameID')->toArray();
```

Comment:
Same as below ad `->toArray()`

Reply:
This one is required, because $oldGames is also passed to `implode()`:
```
implode(): Argument #2 ($array) must be of type ?array, Illuminate\Support\Collection given
```

Reply:
can be done with collections, too, e.g. `$oldGames->implode(',')`